safe-global/safe-wallet-web
# Combine safe lists

## What it solves

Resolves #3418

## How this PR fixes it
- combines the watchlist and owned safes into one list. 
- allows users to bookmark and un-bookmark safes in their list. Bookmarked safes are stored locally and will be displayed also when there is no external wallet connected.

## How to test it
- Open the safe list either in the accounts page of the sidebar of the dashboard with a wallet connected.
- Safes that were added previously should be shown as bookmarked
- Bookmark a safe. Disconnect from your wallet. The bookmarked safes should still be shown (with read-only tag). The rest should not be shown. 
- Click to un-bookmark a safe. It should be removed from the list. 

## Screenshots
connected:
<img width="1512" alt="image" src="https://github.com/safe-global/safe-wallet-web/assets/17801424/b857e858-db2a-4d27-a388-7be21d43650a">

disconnected:
<img width="1512" alt="image" src="https://github.com/safe-global/safe-wallet-web/assets/17801424/12b05f8e-5487-4e28-a923-8fab7693ffef">



## Checklist
